src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -94,6 +94,7 @@ struct sd_dhcp_client {
         size_t client_id_len;
         char *hostname;
         char *vendor_class_identifier;
+        char *user_class;
```

Comment:
I think we should not change the `dhcp_option_append` , rather when we fill up each user class conf we need to split there and append.